nova_vm/src/engine/bytecode/vm.rs

```diff
@@ -418,17 +420,66 @@ impl Vm {
             Instruction::LessThan => {
                 let lval = vm.stack.pop().unwrap();
                 let rval = vm.result.take().unwrap();
-                let result = is_less_than::<true>(agent, lval, rval)
-                    .unwrap()
-                    .unwrap_or_default();
+                let result = match is_less_than::<true>(agent, lval, rval)? {
+                    Some(true) => true,
+                    _ => false,
+                };
+                vm.result = Some(result.into());
+            }
+            Instruction::LessThanEquals => {
+                let lval = vm.stack.pop().unwrap();
+                let rval = vm.result.take().unwrap();
+                let result = match is_less_than::<false>(agent, rval, lval)? {
+                    Some(true) | None => false,
```

Comment:
**nitpick:** This could be done in the same vein as above: `Some(false) => true, _ => false`. Though admittedly, that can also be kind of confusing to read.

Actually, isn't it all just doable as `call(agent, lval, rval)? == Some(cond)`?